src/components/ebay-area-chart/component.js

```diff
@@ -0,0 +1,282 @@
+import Highcharts from 'highcharts';
+import accessibility from 'highcharts/modules/accessibility';
+import patternFill from 'highcharts/modules/pattern-fill';
+import {
+    backgroundColor,
+    gridColor,
+    labelsColor,
+    legendColor,
+    legendInactiveColor,
+    legendHoverColor,
+    tooltipBackgroundColor,
+    tooltipShadows,
+    setSeriesColors,
+} from '../../common/charts/colors';
+
+import { ebayLegend } from '../../common/charts/legend';
+
+if (typeof Highcharts === 'object') {
+    accessibility(Highcharts);
+    patternFill(Highcharts);
+    ebayLegend(Highcharts);
+}
+const pointSize = 1.5;
+
+export default class {
+    onCreate() {
+        this.chartRef = null;
+        this.axisTicksLength = -1;
+        this.state = {
```

Comment:
Do not assign something to state if we are not changing it
Use either `this` or set it as a getter or something